VOKMockUrlProtocol.m

```diff
@@ -23,7 +23,7 @@
     #endif
 #endif
 
-static NSString *const MockDataDirectory = @"VOKMockData";
+static NSString *const DefaultMockDataDirectory = @"VOKMockData";
```

Comment:
how about exporting this publicly?
that would allow us to reset back to the default and create subdirectories and keep things organized.